prow/flagutil/github.go

```diff
@@ -49,6 +52,7 @@ func (o *GitHubOptions) AddFlagsWithoutDefaultGitHubTokenPath(fs *flag.FlagSet)
 }
 
 func (o *GitHubOptions) addFlags(wantDefaultGitHubTokenPath bool, fs *flag.FlagSet) {
+	fs.StringVar(&o.base, "github-base", github.DefaultBase, "GitHub's default base endpoint (may differ for enterprise)")
```

Comment:
I think we should call this the host, not the base

Reply:
Sure, let me name it as `github-host`.

Reply:
Done